components/style/values/specified/basic_shape.rs

```diff
@@ -756,21 +756,21 @@ impl Parse for BorderRadius {
 
 fn parse_one_set_of_border_values(context: &ParserContext, mut input: &mut Parser)
                                  -> Result<[LengthOrPercentage; 4], ()> {
-    let a = try!(LengthOrPercentage::parse(context, input));
+    let a = try!(LengthOrPercentage::parse_non_negavite(input));
```

Comment:
aah, i see. But why is that? Most of the `Parse` in values/* are not actually using context anyways